.github/actions/release-chart-action/cr.sh

```diff
@@ -0,0 +1,249 @@
+#!/usr/bin/env bash
+
+# Copyright The Helm Authors
```

Comment:
I am not sure how to indicate here that the original was authored by somebody else and it was modified.
Also, how do we do opensource licensing in Similarweb?